### ISSUE TITLE
Default page "Remember last page" should include podcast pages

### ISSUE THREAD

**Issue by summersab** [XilinJia/Podcini#66]
### Checklist

- [X] I have used the search function for [**OPEN**](https://github.com/XilinJia/PodVinci/issues) issues to see if someone else has already submitted the same feature request.
- [X] I have **also** used the search function for [**CLOSED**](https://github.com/XilinJia/PodVinci/issues?q=is%3Aissue+is%3Aclosed) issues to see if the feature was already implemented and is just waiting to be released, or if the feature was rejected.
- [X] I will describe the problem with as much detail as possible.
- [X] This request contains only one single feature, **not** a list of multiple (related) features.

### App version

5.5.2

### Where did you get the app from

F-Droid

### Problem you may be having, or feature you want

When the "Default page"  is set to "Remember last page," this only applies to the app's internal pages (Subscriptions, Queue, Downloads, etc). It does not apply to a specific podcast's page.

Here is a use case. Sometimes before I leave the house, I download a few podcast episodes. I open the podcast's page/feed and click download. While I wait for them to finish, I switch to another app. Then, I want to go look at the download progress, so I switch back to Podcini. Instead of the app still being on the podcast's page where I can see if the episodes have finished downloading, I am taken back to the Subscriptions page. I have to either open the podcast's page again or go to the Downloads page.

### Suggested solution

It would be nice if Podcini would remember that a podcast's page was open and have it still be in view after sending Podcini to the background and then reopening it. If it's terribly difficult to implement this, it's only a minor annoyance... but it IS an annoyance. ;-)

**Comment by summersab** [comment]
I _think_ you fixed this? Right now in 6.1.2, it works exactly the way I would like.

I have the default page set to "Subscriptions." That is the page that opens when:
- Podcini is first launched 
- Navigating "upward" through the app's pages by pressing the "back" button

However, Podcini now remembers the app state and will resume the previous view. For example when I do this:
1. Open a subscription's episode list view
2. Send Podcini to the background by pressing the home button or switching to another app
3. Re-open Podcini (either from the drawer/launcher or using the recent app switcher)

Podcini will remember the app state, and the view in point 1 will be restored. AntennaPod and earlier versions of Podcini used to force the default page every time the app was relaunched or sent to the background/brought to the foreground. That was annoying. If the current behavior is the behavior you intended, I'll go ahead and close this issue.

In summary, unless something else is broken . . . leave this alone. It works great! :smile:

**Comment by summersab** [comment]
If you look at how AntennaPod works, it will force the default page even if the app was temporarily sent to the background. I believe early versions of Podcini worked this way, as well. It was extremely annoying - switching to another app such as quickly replying to reply to a text message meant that AntennaPod would always switch to the default page when bringing it to the foreground.

Whatever you did in Podcini to allow secondary pages to be cached and remembered instead of always forcing the default page when bringing the app to the foreground (without the app being killed or closed), it's perfect. Don't change it. :smile:
